src/render/renderer.rs

```diff
@@ -1,16 +1,16 @@
 use image::{ImageBuffer, Rgba};
+use imgui_smithay_renderer::Renderer;
 use smithay::{
     backend::renderer::{
         gles2::{Gles2Error, Gles2Frame, Gles2Renderer, Gles2Texture},
         Frame, Transform,
     },
     utils::{Buffer, Physical, Point, Rectangle},
 };
-
 pub struct RenderFrame<'a> {
     pub transform: Transform,
     pub frame: &'a mut Gles2Frame,
-    pub imgui: &'a imgui::Ui<'a>,
+    pub imgui: Option<(imgui::Ui<'a>, &'a Renderer, Transform)>,
```

Comment:
Isn't this Transform just a duplicate of the transform in the first field? :thinking:

Reply:
sure it is, how have I overlook that 🤔 
fixed in the last commit :)